server/routes/getKeywords.js

```diff
@@ -0,0 +1,7 @@
+const { selectKeywords } = require('../models/queries');
+
+module.exports = (req, res, next) => {
+  selectKeywords()
+    .then(records => records.map(record => record.get('Name')).filter(e => e))
```

Comment:
just a minor tip, to make it more explicit what's the purpose of the last `filter()`, you can write it as `e => !!e`. This will make it clear to the reader that you're evaluating the element for true/false, `e => e` is correct, but looks suspicious at first sight and made me stop and think..
Btw, can you even get records with empty Name field? Don't know how the airtable schema looks or their API works, but maybe it could be configured so that 'Name' is required..

Reply:
thanks